source/Walkthrough/virtual-networks.html.md.erb

```diff
@@ -238,6 +244,10 @@ Navigate to [sandbox.yml](https://github.com/hmcts/azure-public-dns/blob/master/
 ```yaml
   cname:
```

Comment:
They will come under different yaml headers in the file, I would separate them within the example to be completely clear.